src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Browser.cs

```diff
@@ -35,5 +36,10 @@ public Task ConnectAsync(Uri uri, CancellationToken cancellationToken, ClientWeb
             WebSocket = ws;
             return ws.ConnectAsync(uri, options.RequestedSubProtocols, cancellationToken);
         }
+
+        public Task ConnectAsync(Uri uri, HttpMessageInvoker handler, CancellationToken cancellationToken, ClientWebSocketOptions options)
+        {
+            throw new NotImplementedException();
```

Comment:
Can this just call the other `ConnectAsync` and ignore the handler? It would be very convenient for library authors to not need yet another branch for browser specific code.